tests/LSP20CallVerification/LSP6/Interactions/ERC725XExecuteBatch.test.ts

```diff
@@ -0,0 +1,411 @@
+
+      const values = [0, 0, 0];
+
+      const payloads = [lyxDaiTransferPayload, metaCoinTransferPayload, rLYXTransferPayload];
+
+      await universalProfile
+        .connect(context.mainController)
+        .executeBatch(operationTypes, targets, values, payloads);
+
+      expect(await lyxDaiToken.balanceOf(recipient)).to.equal(
+        recipientLyxDaiBalanceBefore.add(lyxDaiAmount),
+      );
+      expect(await metaCoin.balanceOf(recipient)).to.equal(
+        recipientMetaCoinBalanceBefore.add(metaCoinAmount),
+      );
+      expect(await rLyxToken.balanceOf(recipient)).to.equal(
+        recipientRLyxBalanceBefore.add(rLyxAmount),
+      );
+    });
+
+    it('should 1) deploy a LSP7 Token (as minimal proxy), 2) initialize it, and 3) set the token metadata', async () => {
+      const lsp7MintableBase = await new LSP7MintableInit__factory(context.accounts[0]).deploy();
+
+      const lsp7TokenProxyBytecode = String(
+        '0x3d602d80600a3d3981f3363d3d373d3d3d363d73bebebebebebebebebebebebebebebebebebebebe5af43d82803e903d91602b57fd5bf3',
+      ).replace('bebebebebebebebebebebebebebebebebebebebe', lsp7MintableBase.address.substring(2));
+
+      const futureTokenAddress = await context.universalProfile
+        .connect(context.mainController)
+        .callStatic.execute(
+          OPERATION_TYPES.CREATE,
+          ethers.constants.AddressZero,
+          0,
+          lsp7TokenProxyBytecode,
+        );
+
+      const futureTokenInstance = new LSP7MintableInit__factory(context.accounts[0]).attach(
+        futureTokenAddress,
+      );
+
+      const lsp7InitializePayload = futureTokenInstance.interface.encodeFunctionData('initialize', [
+        'My LSP7 UP Token',
+        'UPLSP7',
+        context.universalProfile.address,
+        false,
+      ]);
+
+      // use interface of an existing token contract
+      const tokenMetadataValue =
+        '0x6f357c6aba20e595da5f38e6c75326802bbf871b4d98b5bfab27812a5456139e3ec087f4697066733a2f2f516d6659696d3146647a645a6747314a50484c46785a3964575a7761616f68596e4b626174797871553144797869';
+
+      const lsp7SetDataPayload = futureTokenInstance.interface.encodeFunctionData('setData', [
+        ERC725YDataKeys.LSP4['LSP4Metadata'],
+        tokenMetadataValue,
+      ]);
+
+      const tx = await context.universalProfile
+        .connect(context.mainController)
+        .executeBatch(
+          [OPERATION_TYPES.CREATE, OPERATION_TYPES.CALL, OPERATION_TYPES.CALL],
+          [ethers.constants.AddressZero, futureTokenAddress, futureTokenAddress],
+          [0, 0, 0],
+          [lsp7TokenProxyBytecode, lsp7InitializePayload, lsp7SetDataPayload],
+        );
+
+      // CHECK that token contract has been deployed
+      await expect(tx)
+        .to.emit(context.universalProfile, 'ContractCreated')
+        .withArgs(
+          OPERATION_TYPES.CREATE,
+          ethers.utils.getAddress(futureTokenAddress),
+          0,
+          ethers.utils.hexZeroPad('0x00', 32),
```

Comment:
What is this argument again? Salt I think?

```suggestion
          ethers.utils.hexZeroPad('0x00', 32), // salt
```